Jenkinsfile_nightly

```diff
@@ -26,9 +48,30 @@ def secrets = [
 ]
 
 withNightlyPipeline(type, product, component) {
-    enableFortifyScan()
+  env.NIGHTLY_BUILD = true
+  env.YARN_ENABLE_IMMUTABLE_INSTALLS = false
+  env.TEST_URL = params.CIVIL_SERVICE_URL
+  env.URL = params.FRONTEND_URL
+  env.DM_STORE_URL=params.DM_STORE_URL
+  env.IDAM_API_URL=params.IDAM_API_URL
+  env.SERVICE_AUTH_PROVIDER_API_BASE_URL = params.SERVICE_AUTH_PROVIDER_API_BASE_URL
+  env.CCD_DATA_STORE_URL = params.CCD_DATA_STORE_URL
+  env.CIVIL_SERVICE_URL = params.CIVIL_SERVICE_URL
+  env.CIVIL_GENERAL_APPLICATIONS_URL = params.CIVIL_GENERAL_APPLICATIONS_URL
+  env.WAIT_FOR_TIMEOUT_MS = params.WAIT_FOR_TIMEOUT_MS
+  env.URL_FOR_SECURITY_SCAN = params.CIVIL_SERVICE_URL
+  env.TESTS_FOR_ACCESSIBILITY = false
+  overrideVaultEnvironments([
+    'aat': params.ENVIRONMENT
+  ])
+  loadVaultSecrets(secrets)
+  enableFullFunctionalTest(200)
```

Comment:
do we need cross browser test added as a separate ticket or as part of this ticket ?

Reply:
enabling cross browser as part of different ticket @vasudevganesanhmcts